### ISSUE TITLE
when use img condition，set  do_rgbd_loss = False ，why the result is nothing？<title>

### ISSUE THREAD

**Issue by No360201** [ashawkey/stable-dreamfusion#245]
### Description

zero123 use sjc loss，but in this repo we use sds loss（img condition）.when i set do_rgbd_loss = False, the result is nothing.

### Steps to Reproduce

use img condition ,and set do_rgbd_loss = False

### Expected Behavior

why the result is nothing

### Environment

torch=1.8.0,python=3.7,cuda=11.6

**Comment by voletiv** [comment]
Without an rgbd_loss from the img condition, and without any text prompt, the results would be nothing.
